### PR TITLE
feat: add information about prosecution in Hannover, Germany

### PR DESCRIPTION
**Pull request by levino**
So finally I was able to implement [what I came here to do](https://github.com/weg-li/weg-li/pull/705/commits/8f9f919e2edf5ea3439dd49143d434d5beca7e00). This is a work in progress PR which I will put to "ready for merge" once the Stadt Hannover answers my request. I want to keep this draft PR here so others can see, that I have put in the request for Hannover. I hope it is not too annoying to have lying around.

This PR is based on #702, #696, #698 and #699. It also contains a [commit](https://github.com/weg-li/weg-li/pull/705/commits/82ed30dcff1fa6649be9e7ff09c1494573e4e6b3) that introduces pre-commit hooks via husky so one does not check in offensive code by accident. When doing this, I also fixed some more linting issues in JavaScript and JSON files without changing the ast.

### CODE REVIEW

File: imagine/index.js [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
-  event.respondWith(handleRequest(event.request));
-});
-
-function objectToBase64url(payload) {
-  return arrayBufferToBase64Url(
-    new TextEncoder().encode(JSON.stringify(payload))
-  );
-}
-
+/* globals JWKN ISS JWKE JWKD JWKP JWKQ JWKDP JWKDQ JWKQI BUCKET_NAME */
```
Comment (levino) [inline imagine/index.js:1]: This is very funky. Are these variables really existent in the environment of the cloudflare worker? If not below code seems to be buggy / not doing what it is supposed to do to me.

File: imagine/index.js [hunk 2]
```diff
@@ -116,3 +113,8 @@ async function handleRequest(request) {
     });
   }
 }
+
+/* eslint-disable-next-line no-restricted-globals -- This is just how these webworkers are executed. Cannot change it. */
+addEventListener("fetch", (event) => {
+  event.respondWith(handleRequest(event.request));
+});
```
Comment (levino) [inline imagine/index.js:120]: To satisfy our eslint rules, this must move down in the file. Does not change the logic though.

Comment (phoet) [reply imagine/index.js:120]: Please put that folder into the ignore list, I'm moving this code out of the repo

Comment (levino) [reply imagine/index.js]: Done.

**Comment by phoet** [comment]
if you want this to get merged any time soon, put https://github.com/weg-li/weg-li/pull/705/commits/8f9f919e2edf5ea3439dd49143d434d5beca7e00 in only.

**Comment by levino** [comment]
It will take a while for the answer to come from the city of Hannover. I hope that we can quickly agree on the open questions in the other PRs and when the answer comes back, this PR contains of only one commit.

**Comment by spiii** [comment]
Hey @levino it looks like they won't respond to your request on FragDenStadt.de. In my experience, they don't even reply to emails sent through weg.li with the file reference number. When I asked for the reference numbers, they told me they have too few people and won't do it. This means you should probably finalize your pull request without the response.
As far as I'm aware, Hannover doesn't have an Informationsfreiheitsgesetz (IFG) it seems to be no hope on that.

**Pull request closed without merging**